ScriptingExample/src/test/java/org/openzen/zenscript/scriptingexample/tests/actual_test/arrays/ArraysWithStdLibTests.java

```diff
@@ -22,10 +22,26 @@ public List<String> getRequiredStdLibModules() {
 	}
 
 	@Test
-	public void reverseWorks() {
+	public void reverseWorks_integers() {
```

Comment:
📔 Note: I split this into two test cases because while it doesn't work for primitive base types it does work for Object types like strings.